### PR TITLE
18 display rpaf code

### LINKED ISSUE #18: Display generated .robot code
As a Developer, I'd like to be able to see the code generated in the app, in order to understand what's happening.

---

### :thought_balloon: Hints & Thoughts
If possible we want to [Code Jar](https://github.com/antonmedv/codejar) as it is very lightweight and seems to offer good documentation on the API it offers.
Additionally, there is a react component that is already ready to use.

Please also implement PrismJS syntax highlighting for robot framework.

### :question: Additional Questions to Answer / Consider

---

## :white_check_mark: Conditions of satisfaction

- [ ] Code should be visible with only one click from the modeling scape
- [ ] Code should be runnable as .robot file
- [ ] Code must be correct

### PR DESCRIPTION
**Pull request by kej-jay**
closes #18

## Changes
upon switching to the robot file tab in the web app there now is a code editor present which displays the code generated from the ssot in the database. 
Additionally there is a button to save changes to the cloud which can be used in the future to parse the code and upset it into the database. Currently the method for this already exists, but will only log the robotId and code to the console to display whoever is working on the #190 which variables can be used and where it should be implemented.

## Screenshots
image

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

File: client/src/components/pages/RobotFile/RobotFile.jsx [hunk 1]
```diff
@@ -1,21 +1,79 @@
...
+
+  /**
+   * @description Gets called when the the button is pressed to save to the cloud.
+   * This function will retrieve the code from the editor, parse it to a ssot and write the
+   * resulting ssot into the sessionStorage.
+   */
+  const onSaveToCloud = async () => {
+    const robotId = JSON.parse(sessionStorage.getItem('robotId'));
+    console.log(robotId);
+    console.log(code);
```
Comment (SanJSp) [inline client/src/components/pages/RobotFile/RobotFile.jsx:41]: maybe to fix the linter issues:
```suggestion
    console.warn(robotId,code);
```

Comment (kej-jay) [reply client/src/components/pages/RobotFile/RobotFile.jsx:41]: But it is not really a warning, so I wouldn't want to use that.

**Review by SanJSp — CHANGES_REQUESTED** [review]
Code wise it looks fine. But in the issue it is stated, that the code should not be editable. Are you aware of that?

**Comment by kej-jay** [comment]
> Code wise it looks fine. But in the issue it is stated, that the code should not be editable. Are you aware of that?

Because of the Parser connection issue #1990 I built it so that it could be edited and also added the button already those were not required, but knowing that it will be used in a matter of days or even only hours I added them.

**Review by SanJSp — APPROVED** [review]

**Review by WolfgangDaniel — APPROVED** [review]

**Pull request merged**